doc/vf2_sub_graph_iso.html

```diff
@@ -44,7 +44,8 @@ <h1>
           typename SubGraphIsoMapCallback&gt;
 bool vf2_subgraph_iso(const GraphSmall&amp; graph_small,
                       const GraphLarge&amp; graph_large,
-                      SubGraphIsoMapCallback user_callback)
+                      SubGraphIsoMapCallback user_callback,
+                      bool(*user_step_callback)() = &vf2_trivial_step_callback)
```

Comment:
Instead of being a concrete function pointer, this should be a template like most other parameters. That way, users can pass in whatever they like: lambda, function object, etc.